release-notes/opensearch-alerting-dashboards-plugin.release-notes-2.0.0.0-rc1.md

```diff
@@ -0,0 +1,28 @@
+## Version 2.0.0.0-rc1 2022-04-25
+Compatible with OpenSearch Dashboards 2.0.0-rc1
+
+### Enhancements 
+* Implemented UX support for configuring doc level monitors. ([#218](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/218))
+* Integrate Alerting Dashboards with Notifications Plugin ([#220](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/220))
+* Remove disabled buttons and update Destination flows to reflect read-only state ([#221](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/221))
```

Comment:
wondering if this should go as part of ``Refactor``?

Reply:
I meant for #221 only

Reply:
done